src/App.js

```diff
@@ -18,6 +20,11 @@ const App = (props) => {
         insights.chrome.init();
         insights.chrome.auth.getUser().then(() => setAuth(true));
         insights.chrome.identifyApp('advisor');
+        insights.chrome?.globalFilterScope('insights');
```

Comment:
running locally, using prod-stable things, seeing `Uncaught TypeError: _insights$chrome.globalFilterScope is not a function` n it blows up 

thinking this might be a reason not to merge this in quite yet, cuz its what we might see on prod-stable?